package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-radio-button",
-  "version": "1.2.4",
+  "version": "1.2.5",
```

Comment:
Don't change this as part of this PR - I may include other work in the next release.